.github/workflows/cargo-semver-checks.yml

```
@@ -1,9 +1,19 @@
-    paths: ['**/Cargo.toml']
-  workflow_dispatch:
+    branches: [main, master, develop]
+
+name: cargo-semver-checks
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (4 > 2)

(empty-lines)